e2e/appActions.js

```diff
@@ -192,6 +192,28 @@ async function setTimeConductorMode(page, isFixedTimespan = true) {
     }
 }
 
+/**
+    * Creates a folder in the default location
+    * @param {import('@playwright/test').Page} page - page to load
+    * @param {string} name - the name of the folder
+    */
+async function createFolderInDefaultLocation(page, name) {
```

Comment:
@scottbell this might be too close to using the existing `createDomainObjectWithDefaults` when specifying the folder object.